source/strings.tex

```diff
@@ -5515,38 +5515,38 @@
+  char* strncat(char* s1, const char* s2, size_t n);                    // freestanding
+  int memcmp(const void* s1, const void* s2, size_t n);                 // freestanding
+  int strcmp(const char* s1, const char* s2);                           // freestanding
   int strcoll(const char* s1, const char* s2);
-  int strncmp(const char* s1, const char* s2, size_t n);
+  int strncmp(const char* s1, const char* s2, size_t n);                // freestanding
   size_t strxfrm(char* s1, const char* s2, size_t n);
-  const void* memchr(const void* s, int c, size_t n);           // see \ref{library.c}
-  void* memchr(void* s, int c, size_t n);                       // see \ref{library.c}
-  const char* strchr(const char* s, int c);                     // see \ref{library.c}
-  char* strchr(char* s, int c);                                 // see \ref{library.c}
-  size_t strcspn(const char* s1, const char* s2);
-  const char* strpbrk(const char* s1, const char* s2);          // see \ref{library.c}
-  char* strpbrk(char* s1, const char* s2);                      // see \ref{library.c}
-  const char* strrchr(const char* s, int c);                    // see \ref{library.c}
-  char* strrchr(char* s, int c);                                // see \ref{library.c}
-  size_t strspn(const char* s1, const char* s2);
-  const char* strstr(const char* s1, const char* s2);           // see \ref{library.c}
-  char* strstr(char* s1, const char* s2);                       // see \ref{library.c}
-  char* strtok(char* s1, const char* s2);
-  void* memset(void* s, int c, size_t n);
+  const void* memchr(const void* s, int c, size_t n);                   // freestanding; see \ref{library.c}
+  void* memchr(void* s, int c, size_t n);                               // freestanding; see \ref{library.c}
+  const char* strchr(const char* s, int c);                             // freestanding; see \ref{library.c}
+  char* strchr(char* s, int c);                                         // freestanding; see \ref{library.c}
+  size_t strcspn(const char* s1, const char* s2);                       // freestanding
+  const char* strpbrk(const char* s1, const char* s2);                  // freestanding; see \ref{library.c}
+  char* strpbrk(char* s1, const char* s2);                              // freestanding; see \ref{library.c}
+  const char* strrchr(const char* s, int c);                            // freestanding; see \ref{library.c}
+  char* strrchr(char* s, int c);                                        // freestanding; see \ref{library.c}
+  size_t strspn(const char* s1, const char* s2);                        // freestanding
+  const char* strstr(const char* s1, const char* s2);                   // freestanding; see \ref{library.c}
+  char* strstr(char* s1, const char* s2);                               // freestanding; see \ref{library.c}
+  char* strtok(char* s1, const char* s2);                               // freestanding
```

Comment:
`strtok` is now removed from freestanding library contents of C23 (see [this document](https://docs.google.com/document/d/1S-rgdTiqQ-vQkmxnwZm9JxLBFG2_Wr-3/edit)). Perhaps we need an LWG issue to strike `strtok` and `wcstok`.

Reply:
This is being handled by a future paper.